workers/data_refinery_workers/processors/no_op.yml

```diff
@@ -0,0 +1,30 @@
+---  # "Submitter-processed" processor
```

Comment:
Because the gene identifier conversion for submitter-processed Affymetrix data is done with mappings we've put together in [AlexsLemonade/identifier-refinery](https://github.com/AlexsLemonade/identifier-refinery) and we get those mappings from Zenodo when we build this image: https://github.com/AlexsLemonade/refinebio/blob/3939050215468cf517680119f2c521d127de6fc5/workers/dockerfiles/Dockerfile.no_op#L43

Is it possible to keep track of the version of the Zenodo record?

Reply:
So in this case you want to keep track of `1322711`?

Reply:
I think you'd want to do that with a `cmd_line` specification. Same with the git hash tracking above.

Reply:
@cgreene: I talked to @jaclyn-taroni about the best way to retrieve the version `1322711`. The best approach I can think of is to read `Dockerfile.no_op` and parse that `curl ...` line. Is there any other way?

Reply:
Can you make the Dockerfile have that as a settable variable and also dump that variable to a tiny file, which you could then read easily?

Reply:
Also, what's up with the digits around the zip file? Can they be known in advance or calculated.

Reply:
It's a UNIX timestamp of the file creation time, so `1532646318` is `Thursday, July 26, 2018 11:05:18 PM (UTC)`. This number increases with updated releases.

Reply:
If we want to keep track of the digits in zip file as well, we probably should create a field such as `zenodo_url` in YAML file to record the whole URL.

Reply:
It seems like "what version should we use" for the no_op processor should be a variable. If there isn't a way to make it a variable because of the unix timestamp, then I think that URL should be stored somewhere and read from both places. As it is, it's sort of a magic number in the Dockerfile.

Reply:
If it is a variable, it will have to be defined in some env file and passed to `docker build --envfile <filename> ...`. So it will depend on the docker building command, and we can not guarantee whoever builds the docker image will remember this option. I think it is better to have a static one-line `no_op_zenodo_url.txt` file, which will be read by both Dockerfile.no_op and the program that dynamically collects the version info.

Reply:
Works for me 👍

Reply:
check with @Miserlou for thoughts before implementing. Tagging so he gets a ping

Reply:
@dongbohu IRL discussion w/ @cgreene  has come up with an even ~~~lazier~~~ shorter path such that this could be done inside the dockerfile itself (ex:)

```
RUN ID_REFINERY_URL=https://zenodo.org/record/1322711/files/all_1532646318.zip
RUN curl -O $ID_REFINERY_URL
RUN echo $ID_REFINERY_URL > id_refinery_url.txt
```

Reply:
This looks good. I think it will be better to save `id_refinery_url.txt` using absolute path, something like:
```RUN echo $ID_REFINERY_URL > /etc/id_refinery_url.txt```